### DISCUSSION TITLE
Yaml errors with multiple triggers

### DISCUSSION THREAD

**Discussion by craigz** [community/community#25696] [Actions]
<p>I’ve got my first action working and successfully running using a cron schedule.</p>
<p>I’d like it to <strong>also</strong> trigger on pushes, which would making testing much easier than the whole change the cron trigger time, wait for wall clock time, wait for GitHub execution time, rinse and repeat dance. I found the following syntax suggestion as an accepted answer in another thread:</p>
<pre><code class="lang-auto">on:
  push
  schedule:
    - cron: "22 12 * * *"
</code></pre>
<p>however, when adding the new <code>push</code> line, an error marker appears at the end of the <code>schedule</code> line: <code>bad indentation of a mapping entry</code>.</p>
<p>that error is confirmed via a yaml validator, which validates the following:</p>
<pre><code class="lang-auto">on:
  push
schedule:
    - cron: "22 12 * * *"
</code></pre>
<p>however when entering said ‘valid’ yaml, GitHub then flags the schedule keyword with<br>
<code>unknown key schedule</code>.</p>
<p>so, I’m stuck between a rock and a hard place, where the ‘correct’ yaml syntax is illegal actions syntax, however the apparently correct actions syntax is being flagged as bad yaml.</p>
<p>halp!</p>

**Comment by Simran-B** [comment, marked as answer]
<p>You are missing a colon after <code>push</code> to make it a mapping key:</p>
<pre><code class="lang-yaml">on:
  push:
  schedule:
    - cron: "22 12 * * *"
</code></pre>

**Comment by craigz** [comment]
<p>indeed, that was the issue. thank you very much!</p>
